src/bindings.rs

```diff
@@ -622,18 +627,23 @@ impl<'a> OwnedValueRef<'a> {
     pub fn new(context: &'a ContextWrapper, value: q::JSValue) -> Self {
         Self { context, value }
     }
+    pub fn new_dup(context: &'a ContextWrapper, value: q::JSValue) -> Self {
+        let ret = Self::new(context, value);
+        unsafe{dup_value(ret.context.context, ret.value)};
+        ret
+    }
+
+    /// Get the inner JSValue without increasing ref count
+    pub(crate) fn into_inner(&self) -> &q::JSValue {
```

Comment:
`into_*` methods always take ownership and return an owned value.

This can be called `as_inner`.